snapapi.py

```diff
@@ -73,6 +74,13 @@ def get_screenshot(icao):
   #co.add_argument("--delay 5")
   co.add_argument("--headless")
   co.add_argument("--no-sandbox")
+  # kx1t adds per https://stackoverflow.com/questions/48450594/selenium-timed-out-receiving-message-from-renderer
+  co.add_argument("start-maximized")
+  co.add_argument("enable-automation")
+  co.add_argument("--disable-infobars")
+  co.add_argument("--disable-dev-shm-usage")
```

Comment:
I'd put this as an option (see about 8 lines down) but it didn't seem to help. :shrug: